test/db/cmd/cmd_signature

```diff
@@ -397,7 +383,7 @@ aaa
 Fs bins/flirt/elf-x86/libcurl.a.sig
 EOF
 EXPECT=<<EOF
-Found 205 FLIRT signatures via bins/flirt/elf-x86/libcurl.a.sig
```

Comment:
This?

Reply:
same, the match number is correct, the sig & pat file was made by ida